src/behavior.js

```diff
@@ -59,6 +59,10 @@ const Behavior = MarionetteObject.extend({
   destroy() {
     this.stopListening();
 
+    if (this.view) {
```

Comment:
I think is not necessary this check since view property is assigned unconditionally and in other methods we do not check